code/noRAM/app/build.gradle.kts

```diff
@@ -87,4 +87,13 @@ dependencies {
     // Confetti Library
     implementation("nl.dionsegijn:konfetti-xml:2.0.4")
 
+    //Maps API
+    implementation("org.osmdroid:osmdroid-android:6.1.18")
+    //Location Google Play API
+    implementation("com.google.android.gms:play-services-location:21.2.0")
+    //Map osmdroid-geopackage support, I am not ever sure I need this
+    //I am not sure the version number is correct
```

Comment:
can we remove these?

Reply:
I removed them. If we need them I will add them back later